vmdb/config/routes.rb

```diff
@@ -1882,10 +1882,11 @@
   root :to => 'dashboard#login'
 
   # Enablement for the REST API
-  get '/api'           => 'api#show',    :format => 'json'
-  get '/api/*suffix'   => 'api#show',    :format => 'json'
-  match '/api/*suffix' => 'api#update',  :format => 'json', :via => [:post, :put, :patch]
-  match '/api/*suffix' => 'api#destroy', :format => 'json', :via => [:delete]
+  get '/api(/:version)'           => 'api#show',    :format => 'json', :version => /v\d.*/
+
+  get    '/api(/:version)/:collection(/:c_id(/:subcollection(/:s_id)))' => 'api#show',    :format => 'json', :version => /v\d.*/
+  match  '/api(/:version)/:collection(/:c_id(/:subcollection(/:s_id)))' => 'api#update',  :format => 'json', :via => [:post, :put, :patch], :version => /v\d.*/
```

Comment:
will path segments after the :s_id be accessible by the API code ?  Will want to support in the future virtual subcollections where we map by code remaining part of path to virtual columns/reflections.